recipes/tbb/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# clang: error: invalid deployment target for -stdlib=libc++
+export MACOSX_DEPLOYMENT_TARGET=10.7
+
+make
+make test
```

Comment:
This test suite seems super long. Maybe it is appropriate. Then again, if someone can find a shorter test suite that just makes sure it compiled ok, that would be a fine option IMHO.